comictaggerlib/cli.py

```diff
@@ -265,12 +268,12 @@ def create_local_metadata(self, ca: ComicArchive) -> GenericMetadata:
             if ca.has_metadata(style):
                 try:
                     t_md = ca.read_metadata(style)
-                    md.overlay(t_md)
+                    md.overlay(t_md, self.config.internal__load_data_overlay)
                     break
                 except Exception as e:
                     logger.error("Failed to load metadata for %s: %s", ca.path, e)
 
-        # finally, use explicit stuff
+        # finally, use explicit stuff (always 'overlay' mode)
         md.overlay(self.config.Runtime_Options__metadata)
```

Comment:
Set the mode argument explicitly here `mode=comicapi.genericmetadata.OverlayMode.overlay`

Reply:
I tried `mode=comicapi.genericmetadata.OverlayMode.overlay` but `comicapi` isn't imported so I imported `OverlayMode`.